troposphere/static/js/components/projects/detail/details/SharedProjectView.jsx

```diff
@@ -0,0 +1,148 @@
+            var trClass;
+            switch (instanceStatus) {
+                case "active":
+                    trClass = "active";
+                    break;
+                default:
+                    trClass = "transition"; //"";
+                    break;
+            }
+
+            return (
+                <tr key={instanceID} className="card" onClick={() => this.onSelect(instance)} style={{ cursor: "pointer" }}>
+                    <td>
+                    <Link to={`/projects/shared/instances/${instance.id}`}>
+                                    <Gravatar hash={instanceHash} size={iconSize} type={type} />
+                                   { instanceName }
+                    </Link>
+                    </td>
+                    <td>
+                       { instanceOwner }
+                    </td>
+                    <td className="u-noWrap">
+                       <span
+                           className={`instance-status-light ${trClass}`}
+                       />
+                       { instanceStatus }
+                    </td>
+                    <td>
+                       { instanceIP }
+                       <div className="u-noWrap">
+                           { ( instanceIP !== "N/A" && instanceIP !== "0.0.0.0" ) ?
+                               <CopyButton text={ instanceIP }/> : null
+                           }
+                       </div>
+                    </td>
+               </tr>
+            );
+        }.bind(this));
+        return (
+            <table className="image-requests cy-Table table clearfix">
+                <tbody>
+                    <tr className="card">
+                        <th>
+                            Instance Name
+                        </th>
+                        <th>
+                            Shared By
+                        </th>
+                        <th>
+                            Status
+                        </th>
+                        <th>
+                            IP Address
+                        </th>
+                    </tr>
+                    {instanceRows}
+                </tbody>
+            </table>
+        );
+    },
+    render: function() {
+        var { HelpLinkStore } = this.props.subscriptions,
+            helpLinks = HelpLinkStore.getAll();
+
+        if (!helpLinks) {
+            return (<div className="loading" />);
+        }
+        let instanceSharingLink = HelpLinkStore.get("instance-sharing") || HelpLinkStore.get('request-image'); //FIXME: remove || later.
+
+        return (
+        <div className="container">
+            <p style={{ marginBottom: "16px" }}>
+                {"Looking for more information about the imaging process? Check out the "}
+                <a href={instanceSharingLink.get("href")} target="_blank">documentation on imaging</a>.
+            </p>
+            {this.renderRefreshButton()}
+            {this.renderSharedResourcesTable()}
```

Comment:
Todo: Change the `<p>` at the top to provide help/documentation.

Probably going to want to change the view a bit from what it looks like currently:
<img width="1172" alt="screen shot 2017-11-21 at 3 27 32 pm" src="https://user-images.githubusercontent.com/2889930/33100140-8c71da8a-ced0-11e7-8aac-e48e51869b95.png">